### PR TITLE
upgpkg: rccl 5.2.0-1

### PR DESCRIPTION
**Pull request by acxz**

### CODE REVIEW

**Comment by tpkessler** [comment]
Documentation for rccl can be now found at

https://docs.amd.com/bundle/rccl-release-rocm-rel-5.2/page/library.html

**Comment by tpkessler** [comment]
Please also update the link for `fcf-protection`
https://docs.amd.com/bundle/ROCm-Compiler-Reference-Guide-v5.2/page/Appendix_A.html

File: rccl/PKGBUILD [hunk 1]
```diff
@@ -3,16 +3,16 @@
...
+pkgver=5.2.0
 pkgrel=1
 pkgdesc="ROCm Communication Collectives Library"
 arch=('x86_64')
-url="https://github.com/ROCmSoftwarePlatform/rccl"
+url="https://docs.amd.com/bundle/rccl-release-rocm-rel-5.2/page/library.html"
 license=('custom')
 depends=('hip' 'rocm-smi-lib')
 makedepends=('cmake' 'python' 'gtest')
 source=("$pkgname-$pkgver.tar.gz::$url/archive/rocm-$pkgver.tar.gz")
```
Comment (tpkessler) [inline rccl/PKGBUILD:14]: Set
```
_git='https://github.com/ROCmSoftwarePlatform/rccl'
```
and update the `source` array and the `_dirname` variable.

File: rccl/.SRCINFO [hunk 2]
```diff
@@ -1,6 +1,6 @@
 pkgbase = rccl
 	pkgdesc = ROCm Communication Collectives Library
-	pkgver = 5.1.3
+	pkgver = 5.2.0
 	pkgrel = 1
 	url = https://github.com/ROCmSoftwarePlatform/rccl
```
Comment (tpkessler) [inline rccl/.SRCINFO:5]: Regenerate SRCINFO

**Review by tpkessler — CHANGES_REQUESTED** [review]

**Comment by acxz** [comment]
thx for the thorough review!

**Review by tpkessler — APPROVED** [review]

**Pull request merged**